experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -154,7 +168,13 @@ export interface TreeField extends Tree<FieldSchema>, Iterable<TreeNode> {
 	 */
 	is<TSchema extends FieldSchema>(schema: TSchema): this is TypedField<TSchema>;
 
-	[Symbol.iterator](): Iterator<TreeNode>;
+	/**
+	 * Iterate through all nodes in the field.
+	 *
+	 * @remarks
+	 * This field may not be mutated during iteration.
+	 */
+	boxedIterator(): IterableIterator<TreeNode>;
```

Comment:
this is overriding a member of a base type, and thus does not need its own doc comment unless you need to be more specific.

Reply:
Removed.